src/utils/get-global-config-store.ts

```diff
@@ -41,6 +41,7 @@ export class GlobalConfigStore<
   }
 
   public get(key: string): T[typeof key] {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-return
```

Comment:
This is due to an indirect merge conflict from the rebase on https://github.com/netlify/cli/pull/7165. I had removed a bunch of file-wide eslint suppressions for this file, but that PR reintroduced some. I didn't want to bother digging too much here (`T[type of key]` is `any` by design...).